src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-name/table-interactor-name.component.ts

```diff
@@ -9,22 +8,22 @@ import {organismIcon} from '../../../../../complex-portal-utils';
 })
 export class TableInteractorNameComponent implements OnInit {
 
-  @Input() interactorName: string;
-  @Input() interactorType: string;
-  @Input() interactorId: string;
-  @Input() interactorOrganism: string;
-  @Input() identifierLink: string;
-  @Input() organismIconDisplay: boolean;
-  @Input() interactorTypeDisplay: boolean;
-  @Input() IDDisplay: boolean;
-  @Input() isMainInteractor: boolean;
+  interactorName = input<string>();
+  interactorType = input<string>();
+  interactorId = input<string>();
+  interactorOrganism = input<string>();
+  identifierLink = input<string>();
+  organismIconDisplay = input<boolean>();
+  interactorTypeDisplay = input<boolean>();
+  IDDisplay = input<boolean>();
```

Comment:
idDisplay?

Reply:
Forgot about this one, fixing it

Reply:
Fixed in [6396889](https://github.com/Complex-Portal/complex-portal-view/pull/225/commits/639688955c4b61e90af191972bd83866b8d9044c)